static/app/views/replays/detail/breadcrumbs/useScrollToCurrentItem.tsx

```diff
@@ -34,7 +34,7 @@ function useScrollToCurrentItem({breadcrumbs, ref, startTimestampMs}: Opts) {
   useEffect(() => {
     if (ref.current && current) {
       const index = breadcrumbs?.findIndex(crumb => crumb.id === current.id);
-      ref.current?.scrollToRow(index);
+      ref.current?.scrollToRow(index ? index + 1 : undefined);
```

Comment:
scroll to one extra row, so you can see the purple line clearly basically.

This is for the "next breadcrumb" button, which renders next to the play/pause buttons.